src/main/java/ch/njol/skript/expressions/ExprPing.java

```diff
@@ -32,8 +32,9 @@
 
 @Name("Ping")
 @Description("Returns a player's ping. This expression is only supported on some server software.")
-@Examples({"on join",
-			"\tbroadcast \"%player's ping\""})
+@Examples({"command ping <player=%player%>:",
```

Comment:
You forgot to make the argument optional, also `command /ping` :[ like the current sample scripts lol

Reply:
fixed